build/scripts/meta.yaml.schema

```diff
@@ -17,9 +17,8 @@
       "pattern": "^[-._a-zA-Z0-9]+$"
     },
     "version": {
-      "type": ["number", "string"],
-      "pattern": "^[-._a-zA-Z0-9]+$",
-      "minimum": 0
+      "type": ["string"],
+      "pattern": "^[-._a-zA-Z0-9]+$"
```

Comment:
Dot means any char, right? So, we need escape it.
Does it make sense to deny -_. in the beginning and in the end?

:warning: The proposal is not tested

```suggestion
      "pattern": "^[a-zA-Z0-9]+([-\\._a-zA-Z0-9][a-zA-Z0-9]+)*$"
```

Reply:
Within a character class, `.` has its literal meaning, I believe.

Reply:
@amisevsk You're right. My comment is fixed now.

And what about ._ at the beginning and the end? Does it make sense?

Reply:
Yes sure, I'll add it.